generic_command.py

```diff
@@ -0,0 +1,34 @@
+import asyncio
+
+import iterm2
+
+
+SCRIPT = 'SCRIPT'
+PREFIX = 'PREFIX'
+
+async def main(connection):
+    component = iterm2.StatusBarComponent(
+        short_description='Execute a user defined command',
+        detailed_description='A super simple component where both the command creating the text and a prefix can be defined by the user',
+        exemplar='⎈ docker-for-desktop',
```

Comment:
To match the default values for the knobs:

```suggestion
        exemplar='> hello world',
```